Sources/ContainerClient/Core/Volume.swift

```diff
@@ -0,0 +1,237 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+/// A named volume that can be mounted in containers.
+public struct Volume: Sendable, Codable, Equatable, Identifiable {
+    // id of the volume.
+    public var id: String { name }
+    // Name of the volume.
+    public var name: String
+    // Driver used to create the volume.
+    public var driver: String
+    // The mount point of the volume on the host.
+    public var mountpoint: String
```

Comment:
If I'm following correctly, it's not really a mountpoint, it's just where the block device file is on the host. I'd just call it source.